packages/block-library/src/form-submit-button/edit.js

```diff
@@ -23,7 +23,6 @@ const TEMPLATE = [
 const Edit = () => {
 	const blockProps = useBlockProps();
 	const innerBlocksProps = useInnerBlocksProps( blockProps, {
-		allowedBlocks: TEMPLATE,
```

Comment:
This was not a valid value for `allowedBlocks`, it led to the list being effectively empty and rejecting all child blocks. Added in #44214. @aristath can you confirm that it's correct how I converted the list to `[ 'core/buttons', 'core/button' ]`?

Reply:
Confirmed :+1: